### DISCUSSION TITLE
No mod sliders to define search.

### DISCUSSION THREAD

**Discussion by Krunchy1736** [SnosMe/awakened-poe-trade#809] [Q&A] [answered]
I've been using this tool for several leagues now and I don't remember if it ever let me change values on specific mods when I bring up the search window but recently have seen some videos with a window that looks much different than mine.

The app is up to date. All my settings as far as I know are correct and I'm able to use it for gems, cards, and other currencies. I've exhausted the FAQ, discussions, google searches, a few youtube tutorials on APT, and I can't figure out what's going wrong. 

![image](https://user-images.githubusercontent.com/121326731/209419590-44d25b11-c8bb-40ed-88d8-0f27142d5416.png)

This is what every gear piece, even unique items, brings up. (I have auto search turned off so I don't get locked out from searching on accident.) 
I love this app and will continue to use it but I'm hoping maybe I'm just overlooking something simple.

**Comment by SnosMe** [comment, marked as answer]
Make sure your key to view mods on item in PoE is not assigned to mouse button.
Otherwise you need verify that you can copy Item text with mods info from PoE.
Video https://snosme.github.io/awakened-poe-trade/nothing-happens

**Comment by Krunchy1736** [comment]
Well that  solved it. I swapped it over to M4 as soon as I started playing because I use alt for my second skill bar. Funnily enough in this patch they also separated the highlight and show advanced mod keybind and it appears that the only thing I needed to change was the show advanced mods button off my mouse and it still works. 
Thanks again! That was driving me crazy.
